sim/paladin/retribution/TestShockadin1H.results

```diff
@@ -337,6 +349,9 @@ dps_results: {
 dps_results: {
  key: "TestShockadin1H-Phase7-Lvl60-AllItems-EmeraldEncrustedBattleplate"
  value: {
+  dps: 2357.91835
```

Comment:
There seems to have been some merge conflict in this file that also fails the tests. Duplicate dps/tps entries. This whole file should just be reset in this branch :)